src/qtiCommonRenderer/renderers/interactions/TextEntryInteraction.js

```diff
@@ -72,6 +72,49 @@ function showTooltip($input, theme, message) {
     $input.data('$tooltip').show();
 }
 
+/**
+ * Validate the input for decimal values.
+ *
+ * This function ensures that the input value is either empty or follows
+ * the rules for decimal numbers. It allows numbers with optional
+ * thousands separators (commas) and a mandatory decimal point (dot).
+ *
+ * @param {jQuery} $input
+ */
+function validateDecimalInput ($input) {
+    const separatorName = {
+        '.': __('(dot)'),
+        ',': __('(comma)')
```

Comment:
Would be nice to add `' ': __('(space)')`, because in theory it could be set as thousands separator, and looks weird in that case.